src/eia930.py

```diff
@@ -414,6 +416,8 @@ def manual_930_adjust(raw: pd.DataFrame):
     sc_offsets = (
         raw.index.tz_convert("US/Eastern").to_series().apply(lambda s: s.utcoffset())
     )
+    # After Dec 31, 2020, the offset is 0
+    sc_offsets["2020-12-31T00:00":] = 0
```

Comment:
Does this change it starting on December 31 or on Jan 1, 2021?

Reply:
I updated this to `sc_offsets["2021-01-01T00:00":]` in my most recent commit to start this after Dec 31, but let me know if this is incorrect.

Reply:
Dec 31
![Screen Shot 2022-12-21 at 2 00 35 PM](https://user-images.githubusercontent.com/12755256/209215883-dced0e9a-9e58-4c66-baf6-32130c9d6f72.png)

Reply:
Haha seriously? They couldn't fix this on Jan 1? I'll change back.